modules/services/conky.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+let cfg = config.services.conky;
+
+in with lib; {
+
+  meta.maintainers = [ maintainers.kaleo ];
+
+  options = {
+    services.conky = {
+      enable = mkEnableOption "conky, a light-weight system monitor";
+
+      package = mkPackageOption pkgs "conky" { };
+
+      settings = lib.mkOption {
```

I think this is best changed to be `extraConfig` instead of `settings`. Yeah I know the name is a bit odd considering that there are no other options available but that is the convention.

The reason against `settings` is that it is typically referred to _structural settings_ such as a Nix attribute set serialized to JSON. Not raw text lines, is is the case here.

```suggestion
      extraConfig = lib.mkOption {
```